components/Search/Autocomplete.tsx

```diff
@@ -0,0 +1,195 @@
+    import '@algolia/autocomplete-theme-classic'
```

Comment:
typo. AutoComplete.

Reply:
This is not a typo, that is from the package